### ISSUE TITLE
Is it possible to add and display marked date event data on modal

### ISSUE THREAD

**Issue by harman-brar** [wix/react-native-calendars#564]
## Description

Is it possible to add event data to a marked date in Calendar and then on day press launch a modal that displays this event data?

## Expected Behavior

User presses day on calendar and is shown a modal displaying the event data for that day

## Observed Behavior

```
`_renderDayInfo = (day, onPress) => (
    <View style={styles.mContainer}>
      <Modal
        isVisible={this.state.modalVisible}
        onBackdropPress={() => this.setState({ modalVisible: !this.state.modalVisible })}
        backdropColor={'black'}
        onBackButtonPress={() => this.setState({ modalVisible: !this.state.modalVisible })}>
        <View
          style={styles.bottomModal}>
            <Text style={{ paddingTop: 50 }}>{day}</Text>
            <View
              style={styles.modalBtn}>
                <TouchableOpacity
                  onPress={() => this.setState({ modalVisible: !this.state.modalVisible })}
                  style={styles.btn}>
                    <Text>Close</Text>
                </TouchableOpacity>
            </View>
          </View>
      </Modal>
    </View>
  );

<Calendar>
onDayPress={(day) => this._renderDayInfo(day, this.setState({modalVisible: !this.state.modalVisible}))}
</Calendar>
`

```
```
ReferenceError: ReferenceError: Can't find variable: day

**Comment by cowai** [comment]
@harman-brar Did you find a solution to this? I need the same thing.

**Comment by harman-brar** [comment]
@cowai I created an array of objects of the form:
{ 
"date": "YYYY-MM-DD",
"event": "event name"
}

I stored the day pressed, obtained from the onDayPress handler in a state variable.

Finally, on any given day press I checked if the datestring of my state variable (newly updated by the press) was equal to any found in the array created earlier. If yes, I created a modal using an external library and set the corresponding object's event as the text.

Important: make sure the datestring formats in your array match those given by the onDayPress handler.
